README.md

```diff
@@ -6,10 +6,18 @@
 
 # Parity Signer - Turn your smartphone into a hardware wallet
 
-Parity Signer is a mobile application that turns a smartphone into a secure air-gapped wallet also called cold storage. It means that, once the app has been installed, users can create Ethereum accounts, sign transactions and transfer funds from and to these accounts without any sort of connectivity enabled on the device. Wifi, Mobile Network, Bluetooth can be shut down and even removed physically to ensure that the mobile phone containing these accounts will not be exposed to any online threat. Have a look at the tutorial on our wiki to learn how to use [Parity Signer together with Fether](https://wiki.parity.io/Parity-Signer-Mobile-App-Fether-tutorial) or [MyCrypto app](https://wiki.parity.io/Parity-Signer-Mobile-App-MyCrypto-tutorial).
+Parity Signer is a mobile application that allows any smartphone to act as an air-gapped crypto wallet also known as "cold storage."
+
+Once the app has been installed, users can create Ethereum accounts, sign messages, transactions and transfer funds to and from these accounts without any sort of connectivity enabled on the device.
```

Comment:
suggestion: `...create Ethereum accounts, sign messages/transactions, and transfer funds to and from these...``